.travis.yml

```diff
@@ -1,12 +1,10 @@
 language: scala
 scala:
-   - 2.10.4
```

Comment:
Is it really so? As I understood from https://playframework.com/documentation/2.4.x/Migration24#Java-8-support new projects should set scalaVersion to 2.11.6+